airflow/contrib/hooks/gcp_dataflow_hook.py

```diff
@@ -171,3 +177,17 @@ def _build_cmd(self, task_id, variables, dataflow):
             for attr, value in variables.iteritems():
                 command.append("--" + attr + "=" + value)
         return command
+
+    def _start_template_dataflow(self, name, variables, parameters,
+                                 environment, dataflow_template):
+        body = {"jobName": "{jobname}".format(jobname=name),
+                "parameters": parameters,
+                "environment": environment}
+        service = self.get_conn()
+        request = service.projects().templates().launch(projectId=variables['project'],
```

Comment:
What if the 'project' is not set? Maybe consider adding some input validation logic.